content/blog/simplify-embedded-projects-with-esp-bsp/index.md

```diff
@@ -0,0 +1,131 @@
+
+## What is ESP-BSP?
+
+[ESP-BSP](https://github.com/espressif/esp-bsp) is a collection of Board Support Packages featuring Espressif’s or M5Stack development boards. The set can be extended to any board with Espressif’s chip. It provides a convenient way to start a project for a specific development board without the need to manually look for drivers and other details. By using ESP-BSP, you can:
+
+- **Streamline Hardware Integration**: Simplify code and reduce complexity.
+- **Enhance Development Speed**: Quickly set up and start working on your projects.
+- **Access Standardized APIs**: Ensure consistency across your projects.
+
+## Getting Started with ESP-BSP
+
+The following example covers steps for building application for ESP32-S3-BOX-3 which is supported by ESP-BSP.
+
+### Hardware Setup
+
+Ensure you have the following hardware:
+
+- [ESP32-S3-BOX-3](https://github.com/espressif/esp-box) development board.
+- USB-C Cable for power and programming.
+
+### Prerequisites
+
+Before you begin, make sure you have the following:
+
+- [ESP-IDF v5.3](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/get-started/index.html#installation): The official development framework for the ESP32, properly installed and sourced in your shell.
+
+### Creating Project from Example
+
+Let’s create a simple project using the `display_audio_photo` example, which is available for the ESP32-S3-BOX-3. This example showcases how to use the display, touch, and audio features.
+
+1. **Initialize a New Project**:
+
+   Use the `idf.py` tool to create a new project from the example:
+
+   ```bash
+   idf.py create-project-from-example "espressif/esp-box-3^1.2.0:display_audio_photo"
+   cd display_audio_photo
+   ```
+
+2. **Set the Target**:
+
+   Ensure the correct target is set for your project:
+
+   ```
+   idf.py set-target esp32s3
+   ```
+
+   Note: For users of ESP-IDF 5.3 or newer, it is necessary to add the following dependency with the corrected I2C driver (error message at runtime: `CONFLICT! driver_ng`):
+
+   ```
+   idf.py add-dependency "espressif/esp_codec_dev==1.1.0"
+   ```
+
+3. **Build and Flash the Project**:
+
+   Compile and flash your application to the ESP32-S3-BOX-3:
+
+   ```
+   idf.py build flash monitor
```

Comment:
Consider recording the output with asciinema.

Reply:
Done. @f-hollow The media file is with the article. IMO it's better when article and it's media is collocated, there is already featured.webp, so it kind of make same to have this content there.

I'm ok with moving asciinema to different location, but please explain why the other location should be better.

Reply:
Solved. Assets should be co-located.